### ISSUE TITLE
Thought disorder HP:5200235 definition and/or renaming

### ISSUE THREAD

**Issue by kscott66** [obophenotype/human-phenotype-ontology#10178]
**HPO term**
Thought disorder HP:5200235

Other terms mentioned: 
- Abnormal language feature HP:4000072
- Flight of ideas HP:5200234
- Loose association HP:5200006
- Thought blocking HP:5200227
- Pressured Speech HP:5200265

**Suggested revision and reasons**
For the mania group, it was difficult to find the pressured speech term in the hierarchy. Thought disorder as a parent term was confusing, being under abnormal language feature. Terms currently under thought disorder: Pressured speech, loose association, thought blocking, and flight of ideas may be better suited under a "disorganized speech" or "disorganized communication" parent term. Additionally, the thought disorder term definition is making assumptions of the underlying cause for an observation: "An abnormal or disorganized way of communicating that appears to be the result of disorganized thought."

A change could be renaming the thought disorder term to disorganized speech or communication, and revising the definition (eg. an abnormal or disorganized way of communicating, which may be the result of disorganized thought, distractability, racing thoughts, or other internal experiences). That way, the four current child terms of thought disorder would reasonably fit under the new parent term in the same position of the hierarchy.

**Comment by pnrobinson** [comment]
@bcoleman96 
sorry for the late response. 
This is the current structure [Thought disorder HP:5200235](https://hpo.jax.org/app/browse/term/HP:5200235) 
In general, it is probably better not to use "disorder" as a term label because this sounds more like a disease than a phenotypic feature, and so the idea of labelling "Disorganized speech or communication" (or something like that, sounds good.

Thoughts?

**Comment by pnrobinson** [comment]
@bcoleman96  @kscott66 

I agree with the suggestion by @kscott66 and will rename Thought disorder HP:5200235 as follows unless there are objections.


**Disorganized speech**
**definition**: Speech that is difficult for the listener to understand due to its poor organization and lack of semantic cohesion. 
**comment**: Disorganized speech is a component of Formal thought disorder (FTD), a core symptom of schizophrenia. FTD represents a fundamental disruption in the organization and maintenance of goal-directed thought. In contrast to disordered thought content (e.g., delusions), disorganized speech represents disordered thought form (e.g., derailment and tangentially).
PMID:32200776
**Related synonyms**: Thought disorder, Formal thought disorder

**Comment by bcoleman96** [comment]
@pnrobinson 

Clinically, we would say that each of the children is a type of Thought disorder, so I would at least leave Thought disorder as a synonym to make sure these terms can be found by clinicians. In terms of the definition, I don't think we should require that it is hard for the listener to understand as this is not always the case. I'm also not sure I agree that the original definition assumes causation since it indicates that these are patterns of speech that seem to arise from abnormal patterns of thought. (That's not to say that they are actually arising from abnormal patterns of thought.) I think this aspect is important to include to distinguish things like pressured speech and increased speech rate. I would suggest the following definition:
Disorganized or unusual pattern of speech that appears to arise from abnormal thought patterns.

**Comment by pnrobinson** [comment]
updated label as suggested.
